9c-main/multiplanetary/network/9c-network.yaml

```
@@ -448,3 +448,9 @@ rudolfService:
     securityGroupIds:
     - "sg-0f0bf654f2ff02289"
     - "sg-033602a010bce902e"
+
+testWorldBoss:
+  enabled: true
+
+  nodeSelector:
+    node.kubernetes.io/instance-type: m5.large
```

```suggestion
    eks.amazonaws.com/nodegroup: 9c-main-m5_l_2c
```

So that odin's service doesn't use heimdall's nodegroup.